src/screens/TokenDetail.js

```diff
@@ -21,85 +21,66 @@ import { TX_COUNT } from '../constants';
 class TokenDetail extends React.Component {
   /**
    * token {Object} selected token data
```

Comment:
Can you  describe here the expected keys of the `Object`? When we write 'token data' is usually means an object with {uid, name, symbol} but you are adding lots of information on it right now, right?

Reply:
Yes, @msbrogli said it above and i agree too.